daml-foundations/daml-tools/da-hs-daml-cli/DA/Cli/Damlc.hs

```diff
@@ -98,7 +98,7 @@ cmdTest numProcessors =
   where
     cmd = execTest
       <$> many inputFileOpt
-      <*> fmap (\x -> ColorTestResults x) colorOutput
```

Comment:
Note that HLint finds this with `Ignore: Avoid lambda`, so if we didn't ignore that, we'd have been fine.

Reply:
I did not know that,  I need to learn using HLint